### DISCUSSION TITLE
Possible to add another widget?

### DISCUSSION THREAD

**Discussion by voxvargr** [breezy-weather/breezy-weather#578] [Ideas]
![Screenshot_20231201-160718.png](https://github.com/breezy-weather/breezy-weather/assets/65240716/7a718996-e34c-4829-830c-f78a165fe342)

Would it be possible to add another widget like this one but with another row added to the bottom to be hourly weather to match the daily above it? Also is it possible to move everything up just a bit because if I try to shrink the widget it cuts stuff off but still has the big emptiness at the top.

**Comment by papjul** [comment]
Have a look at Material You - Forecast widget
